pkg/nsx/services/vpc/compare_test.go

```diff
@@ -0,0 +1,8 @@
+package vpc
+
+import (
+	"testing"
+)
+
+func TestIsVPCChanges(t *testing.T) {
```

Comment:
Do you want to add test in this patch?

Reply:
No, i just added an empty file since the file is already created, the test code is in another patch